acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/site/HomePage.java

```diff
@@ -0,0 +1,90 @@
+package uk.nhs.digital.ps.test.acceptance.pages.site;
+
+import org.openqa.selenium.By;
+import uk.nhs.digital.ps.test.acceptance.pages.PageHelper;
+import uk.nhs.digital.ps.test.acceptance.webdriver.WebDriverProvider;
+
+public class HomePage extends AbstractSitePage{
+
+    private PageHelper helper;
+
+
+    public HomePage(WebDriverProvider webDriverProvider, final PageHelper helper) {
+        super(webDriverProvider);
+        this.helper = helper;
+    }
+
+    public String getPageTitle(){
+        return getWebDriver().findElement(By.xpath("html/body/header/div/div/section/div/form/div[2]/button")).getText();
+    }
+
+    public String findPageElement(String elementName) {
+        if(elementName.toLowerCase().equals("top label")) {
```

Comment:
What is the reason for not using data-uipath at this point?

Reply:
No data-uipaths added to free marker templates yet.